commands/Misc/echo.js

```diff
@@ -14,8 +14,11 @@ module.exports = class extends Command {
     }
 
     async run(msg, [channel = msg.channel, ...message]) {
-        if (channel.postable === false) throw msg.language('SYSTEM_CHANNEL_NOT_POSTABLE');
```

Comment:
This does exist.

https://github.com/dirigeants/klasa/blob/628fa2b5d75e677510adb9f83b7ecfefd6677192/src/extendables/MessageExtendables.js#L18-L20

Reply:
```javascript
if (!channel.postable) throw msg.language('SYSTEM_CHANNEL_NOT_POSTABLE');
```

Reply:
Please change the `=== false` to `!`